codeforamerica/vita-min
# Nj 49 line 40b

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/49

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
Add a question for homeowner or renter status. Add it to XML and PDF.

## How to test?
Go through the filing process, selecting any sample directfile data. You should see a question after county and municipality that asks whether you were a renter or a homeowner in the last tax year. The choice you select should show up in the XML and PDF submissions when you are done.

## Screenshots (for visual changes)
![image](https://github.com/user-attachments/assets/bca57d27-2c5a-43ec-9dbc-1c7ad058984c)
![image](https://github.com/user-attachments/assets/f0d7f5b9-1504-4a26-96ce-733b5e7d5e9b)
![image](https://github.com/user-attachments/assets/c7618454-83f1-455a-ba4b-8f7b9720107f)
![image](https://github.com/user-attachments/assets/dd030ea4-944a-4f3a-923e-42f5bacfcb85)
